docs/crds/hostrule.md

```diff
@@ -212,11 +212,36 @@ Where dedicated VSes are created corresponding to a single application, Shared V
         tcpSetting:
           loadBalancerIP: 10.10.10.1
 
+
         fqdn: Shared-VS-L7-1      # bound for clusterName--Shared-VS-L7-1
         fqdnType: Contains
         tcpSetting:
           loadBalancerIP: 10.10.10.1
 
+##### Custom Ports
+
+In order to overwrite the ports opened for VSes created by AKO, users can provide the port details under the `listeners` setting. The ports mentioned under this section are overwrites the default open ports, 80 and 443 (SSL enabled). This is applicable only for Shared or Dedicated virtual services.
```

Comment:
`The ports mentioned under this section **are** overwrites the default open ports`
lets remove **are**